react-native/react/reducers/serialize.mobile.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+/* @flow */
+
+import { AsyncStorage } from 'react-native'
+import type { State } from '../constants/reducer-types'
+import { STATE_KEY } from '../constants/reducer-types'
+import transit from 'transit-immutable-js'
+
+export default function (state: State, action: any): State {
+  if (action.type === 'restoreState') {
```

Comment:
I was being lazy since this is dev only stuff, but I can create a `../constants/dev` file